src/luascript.cpp

```diff
@@ -11550,6 +11551,18 @@ int32_t LuaScriptInterface::luaItemTypeGetDecayId(lua_State* L)
 	return 1;
 }
 
+int32_t LuaScriptInterface::luaItemTypeGetRequiredLevel(lua_State* L)
+{
+	// itemType:getRequireLevel()
+	const ItemType* itemType = getUserdata<const ItemType>(L, 1);
+	if (itemType) {
+		lua_pushnumber(L, itemType->minReqLevel);
+	}else {
```

Comment:
I can agree, I hate using git and I can not make the git software work...
Can't even get it to update here on my repo without it saying I don't own this repo. Idk... tried to google things for weeks with no luck.
But the thing is we should keep using this code format, better then ex. 0.4 where it's all just a big mess (just try to compare reading the protocollogin file or something else that isen't huge)...